hw1/01-fizzbuzz.js

```diff
@@ -1,32 +1,24 @@
-/** Exercise 01 - Fizzbuzz
+/* Exercise 01 - Fizzbuzz
 
 Write a program that writes all the numbers from 1 to 100, with some exceptions: 
 - For numbers divisible by 3, print “fizz” 
 - For numbers divisible by 5 (but not 3), print “buzz” 
 - For numbers divisible by 3 and 5, print “fizzbuzz”
-
+ 
 Use console.log() to write the proper output to the command line.
 
-**/
+*/
 
 const fizzbuzz = () => {
-  // Add your code here
+  for (let i = 1; i <= 100; i += 1) {
+    if (i % 5 === 0 && i % 3 === 0) {
+      console.log(i, "fizzbuzz");
```

Comment:
Looking good! It looks like you are also printing the number along with 'fizz', 'buzz', and 'fizzbuzz' instead of just doing 'fizz', 'buzz', and 'fizzbuzz'.